pkgs/top-level/all-packages.nix

```
@@ -15460,6 +15460,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  buildTypstPackage = callPackage ../build-support/build-typst-package.nix { };
+
```

Did you forget to register `pgs.typstPackage`?

```suggestion

  typstPackages = typst.typstPackages;

```